workshops/book-dash/book-dash-mcr-report.md

```diff
@@ -0,0 +1,162 @@
+# The Turing Way | Book Dash - Manchester
+
+
+* Intro to Book Dash MCR slides: https://github.com/alan-turing-institute/the-turing-way/blob/master/workshops/book-dash/IntroBookDashMCR.pdf
+* HackMD: http://bit.ly/book-dash-mcr
+
+## The participants
+
+![https://pbs.twimg.com/media/D6y7be5XYAAF0i2?format=jpg&name=900x900](https://pbs.twimg.com/media/D6y7be5XYAAF0i2?format=jpg&name=900x900)
+
+* Rachael Ainsworth [@rainsworth](https://github.com/rainsworth/)
+* Tarek Allam Jr [@tallamjr](https://github.com/tallamjr)
+* Tania Allard [@trallard](https://github.com/trallard)
+* Oliver Clark [@OliJimbo](https://github.com/OliJimbo)
+* Alex Clarke [@informationcake](https://github.com/informationcake/)
+* Jez Cope [@jezcope](https://github.com/jezcope)
+* Joe Fennell [@joe-fennell](https://github.com/joe-fennell)
+* Patricia Herterich [@pherterich](https://github.com/pherterich)
+* Rosie Higman [@rosiehigman](https://github.com/rosiehigman)
+* Will Hulme [@wjchulme](https://github.com/wjchulme) 
+* Greg Kiar [@gkiar](https://github.com/gkiar) 
+* Clare Liggins [@ClareLiggins](https://github.com/ClareLiggins)
+* Javier Moldon [@jmoldon](https://github.com/jmoldon)
+* Beth Montague-Hellen [@AlfAWolf140](https://github.com/AlfAWolf140)
+* Jade Pickering [@jspickering](http://www.github.com/jspickering)
+* Rosti Readioff [@RostiReadioff](https://github.com/RostiReadioff)
+* Kirstie Whitaker [@KirstieJane](https://github.com/KirstieJane/)
+* Yo Yehudi [@yochannah](http://www.github.com/yochannah)
+
+
+## Our report
+
+### What did we do?
+
+Our goal for the book dash was to bring together participants enthusiastic about reproducibility to contribute to and improve the *Turing Way* book during a one day collaborative event. 
+We held a networking event the evening prior to the book dash for participants to get to know each other and promote any projects that they're working on, in order to have an intense 9 to 5 day working on the book.
+There was a great diversity of lightning talks which were really fun:
```

Comment:
Needs an extra sentence about the networking event. It wasn't _just_ to make them work harder on the second day.

* ice breaker
* get to know each other
* reward and thanks for taking time out to work on the project
* lightning talks to share experiences and expertise.